tests/mr_test_module/src/lib.rs

```diff
@@ -884,10 +884,11 @@ impl Drop for KeysReader {
 static mut HASH_RECORD_TYPE: Option<RecordType<StringRecord>> = None;
 static mut INT_RECORD_TYPE: Option<RecordType<IntRecord>> = None;
 
-fn init_func(ctx: &Context, _args: &Vec<RedisString>) -> Status {
+fn init_func(ctx: &Context, args: &Vec<RedisString>) -> Status {
     unsafe{
         DETACHED_CTX = RedisModule_GetDetachedThreadSafeContext.unwrap()(ctx.ctx);
-        MR_Init(ctx.ctx as *mut libmrraw::bindings::RedisModuleCtx, 3, RedisModule_StringPtrLen(_args[0], ptr::null_mut()));
+        let passwd = if args.len() > 0 {RedisModule_StringPtrLen(args[0], ptr::null_mut())} else {ptr::null_mut()};
```

Comment:
```suggestion
        let passwd = args.get(0).unwrap_or(ptr::null_mut());
```
`args` is a vector, the get function will return rust `None` is position 0 does not exists, then we will map it to Null.